src/Compilers/CSharp/CSharpAnalyzerDriver/CSharpDeclarationComputer.cs

```diff
@@ -183,7 +183,7 @@ private static bool InvalidLevel(int? level)
                 case SyntaxKind.GetAccessorDeclaration:
                     {
                         var t = (AccessorDeclarationSyntax)node;
-                        builder.Add(GetDeclarationInfo(model, node, getSymbol, t.Body, cancellationToken));
+                        builder.Add(GetDeclarationInfo(model, node, getSymbol, t.Body ?? (SyntaxNode)t.ExpressionBody, cancellationToken));
```

Comment:
>t.Body ?? (SyntaxNode)t.ExpressionBody [](start = 79, length = 38)

Is there a test that demonstrate significance of this change? Also, looking at the code below (for SyntaxKind.MethodDeclaration, etc.), we should probably concatenate Body and ExpressionBody when both are present.

Reply:
@AlekseyTs ```AccessorDeclarationSyntax``` can't have Body and ExpressionBody at the same time.
Regarding to SyntaxKind.MethodDeclaration case I assume that the only one reason of using concatenation is passing list of parameters with Body or ExpressionBody.

Reply:
> AccessorDeclarationSyntax can't have Body and ExpressionBody at the same time.

I think it can. This doesn't mean the program is valid, but it looks like both can present at the same time. You can try to add a test for a method, see what happens and then test similar scenario for an accessor. I would expect to observe similar behavior.